invenio_circulation/api.py

```diff
@@ -101,3 +103,11 @@ def save(self, event):
         else:
             self['state'] = self.state
             self.commit()
+
+    @classmethod
+    def save_diagram(cls, output_file):
```

Comment:
what about `export` or `generate` diagram?

Reply:
Ok go for `export_diagram`